src/lib/UI/Value/Content/ContentDraft.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\UI\Value\Content;
+
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+
+class ContentDraft implements ContentDraftInterface
```

Comment:
IMHO it's an overkill to create wrappers for VOs from Public API. Is it a pattern in AdminUI? Not requesting changes, just asking.